### PR TITLE
Add basic installation instructions to README

### PR DESCRIPTION
**Pull request by jamiesnape**
@cpatrick PTAL.

### CODE REVIEW

**Comment by cpatrick** [comment]
LGTM, but why lowercase?

**Comment by jamiesnape** [comment]
Packages package names are always lowercase.

File: README.md [hunk 1]
```diff
@@ -14,7 +17,43 @@ maintain large-scale software systems. Good examples of a CDash are the
...
+Download and run [Composer-Setup.exe](https://getcomposer.org/Composer-Setup.exe).
+
+```cmd
+git clone https://github.com/Kitware/CDash.git CDash
+cd CDash
+```
+
+For development:
+
+```cmd
```
Comment (billhoffman) [inline README.md]: Need to cd to CDash directory to run composer install

**Comment by cpatrick** [comment]
kk

**Comment by jamiesnape** [comment]
We need to tag a version at some point. It should be `x.y.z` rather than `Release-x-y-z`.

**Pull request merged**